CHANGELOG.md

```diff
@@ -1,12 +1,15 @@
 Scalyr Agent 2 Changes By Release
 =================================
 
-## 2.1.8 "TBD" - July 10, 2020
+## 2.1.8 "Titan" - August 3, 2020
 
 <!---
-Packaged by Steven Czerwinski <czerwin@scalyr.com> on Jul 20, 2020 08:30 -0800
+Packaged by Steven Czerwinski <czerwin@scalyr.com> on Aug 3, 2020 12:30 -0800
 --->
 
+Bug fixes:
+* Fixed race condition in pipelined requests which could lead to duplicate log upload, especially for systems with a large number of inactive log files.  Log files would be reuploaded from their start over short period of time (seconds to minutes).
```

Comment:
Is it also worth adding which versions and configurations would it affect (aka > 2.6.1 or pipeling enabled)?

Reply:
Good point.  I'd added that note to this comment.